tests/files/cycles/flow-types.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+import type { FooType } from './flow-types-depth-two';
```

Comment:
Should this also work for `import { type FooType } ...` syntax?

Reply:
This complicates things a little, since import kind becomes the value. I didn’t discover how to identify such type imports yet